### PR TITLE
add support for loong64

### PR DESCRIPTION
**Pull request by yetist**
Fix the issue that can not be build on loongarch, please review, thank you.

### CODE REVIEW

**Comment by ysicing** [comment]
@hugelgupf @10000TB  

Go programming language support added so recently, can you consider supporting Loongson arch (loong64)

By any chance either of you are free this week to review this? Thank you 🙏🏼

File: cross-compile.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ function buildem() {
   done
 }
 
-GOARCHES="386 amd64 arm arm64 ppc64 ppc64le s390x mips mipsle mips64 mips64le"
+GOARCHES="386 amd64 arm arm64 ppc64 ppc64le s390x loong64 mips mipsle mips64 mips64le"
```
Comment (bradfitz) [inline cross-compile.sh]: I would revert this part. This'll require they use Go 1.19 (not sure if they do). But the other part of the change is safe.

Comment (ysicing) [reply cross-compile.sh]: @yetist PTAL

Comment (bradfitz) [reply cross-compile.sh]: I still see it. Did you forget to push?

Comment (yetist) [reply cross-compile.sh]: Done.

**Comment by abner-chenc** [comment]
Please also update the version of sys,  Thanks.
```
 go get -d golang.org/x/sys@fc697a31fa06b616162e34fd66047ab52722ba6c
 go mod tidy
```

**Comment by yetist** [comment]
> Please also update the version of sys, Thanks.
> 
> ```
>  go get -d golang.org/x/sys@fc697a31fa06b616162e34fd66047ab52722ba6c
>  go mod tidy
> ```

Done.

**Review by 10000TB — APPROVED** [review]

File: cross-compile.sh (hunk 1 above)
Comment (10000TB) [reply cross-compile.sh]: consider resolving this if addressed

**Review by hugelgupf — APPROVED** [review]

**Pull request merged**